store/listings/management/commands/fetch_listings.py

```diff
@@ -37,6 +40,17 @@ def handle(self, *args, **options):
                 'filename': 'cotton_eur.txt'
             },
         ]
-
         for listings in listings_data:
-            save_to_file(listings=listings)
+            for listing in listings['data']:
+                listing_object = Listing.objects.create(
+                    title=listing.get('title'),
+                    description=listing.get('description'),
+                    price=listing.get('price'),
+                    currency=listing.get('currency'),
+                    category=listings['category']
```

Comment:
`category` is not needed here. API response does not include it. Your `Listing` model doesn't have it.